KNM/events/KnightsLanding_matrix.json

```diff
@@ -271,14 +271,14 @@
       "MATRIX_RESPONSE": "L2_HIT_NEAR_TILE",
       "MATRIX_VALUE": "0x1800180000",
       "MATRIX_REGISTER": "0,1",
-      "DESCRIPTION": "accounts for reponses from snoop request hit with data forwarded from its Near-other tile L2 in E/F/M state"
+      "DESCRIPTION": "accounts for responses from snoop request hit with data forwarded from its Near-other tile L2 in E/F/M state"
```

Comment:
Looks good to me. Please increase the version in the header at the top of this file.